backend/gateway/meta/meta.go

```diff
@@ -320,7 +321,9 @@ func APIBody(body interface{}) (*anypb.Any, error) {
 	return anypb.New(ClearLogDisabledFields(m))
 }
 
-/* ToValue converts custom types to a structpb.Value. This helper was added
+/*
+	ToValue converts custom types to a structpb.Value. This helper was added
+
```

Comment:
Some weird indentation going on here

Reply:
Yeah that's a change in `gofmt` in Go 1.18. I didn't make any changes to source code (aside from `ReadHeaderTimeout`). I just upgraded `golangci-lint` and ran `golangci-lint run --fix`.